internal/commands/pull/command.go

```diff
@@ -48,26 +49,48 @@ func (cmd *Command) Flags() []flags.Flag {
 		flags.StringFlag{
 			Value: &cmd.inputs.RemoteApp,
 			Meta: flags.Meta{
-				Name: "remote",
+				Name: flags.FlagRemote,
 				Usage: flags.Usage{
 					Description: "Specify the name or ID of a remote Realm app to export",
 				},
 			},
 		},
+		flags.BoolFlag{
+			Value: &cmd.inputs.IncludeNodeModules,
+			Meta: flags.Meta{
+				Name: flags.FlagIncludeNodeModules,
+				Usage: flags.Usage{
+					Description: "Export and include Realm app dependencies from an archive file",
+					Note:        "The allowed formats are as a directory or compressed into a .zip, .tar, .tar.gz, or .tgz file",
+				},
+			},
+		},
+		flags.BoolFlag{
+			Value: &cmd.inputs.IncludePackageJSON,
+			Meta: flags.Meta{
+				Name: flags.FlagIncludePackageJSON,
+				Usage: flags.Usage{
+					Description: "Include Realm app dependencies in the diff from a package.json file",
+				},
+			},
+		},
+		// TODO(REALMC-10088): Remove this flag in realm-cli 3.x
 		flags.BoolFlag{
 			Value: &cmd.inputs.IncludeDependencies,
 			Meta: flags.Meta{
-				Name:      "include-dependencies",
+				Name:      flags.FlagIncludeDependencies,
```

Comment:
and this one probably doesn't need a `const` defined for it (since we're never really referencing it anywhere else, not to mention it will soon go away entirely)